app/models/batch_upload.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+class BatchUpload < ActiveRecord::Base
+  has_many :uploaded_files
```

Comment:
@RudyOnRails what about calling this `UploadedBatch` or even `UploadedFileBatch`? It might help distinguish it from the other classes that deal with batch edits, etc. like `BatchEditItem` and `BatchUpload`.

Reply:
@awead hmm, I don't see any other Ruby classes named `BatchUpload`. Don't we want to keep it similar to what `Sufia::BatchUploadsController` handles?